tests/Doctrine/Tests/ORM/EntityManagerTest.php

```diff
@@ -204,4 +207,15 @@ public function transactionalCallback($em)
         $this->assertSame($this->_em, $em);
         return 'callback';
     }
+
+    /**
+     * @expectedException \InvalidArgumentException
```

Comment:
@SpacePossum then the new API. We can bump the required PHPUnit version without any issues 👍